src/middleware/authorization-middleware.ts

```diff
@@ -1,14 +1,11 @@
 import { RequestHandler } from 'express';
 import HttpException from '../exceptions/http/http-base-exception';
 import fetch, { Response } from 'node-fetch';
-import config from 'config';
 import jwt_decode from 'jwt-decode';
 import { Forbidden, UnAuthenticated } from '../exceptions/http/http-exceptions';
 import { UserProfile } from '../model/dto/user-profile-dto';
 import { Utility } from '../utility/utility';
-
-const permissionUrl: string = config.get('url.permission');
```

Comment:
should have changed the value here rather than replacing all the occurences in the file

Reply:
We are reading from environment class defined and we read from there. this will be duplicate variable to introduce